api2/pkg/webapp/cases.go

```diff
@@ -196,8 +196,10 @@ func (h *Handler) PostCase(ctx context.Context, id string, w http.ResponseWriter
 	done := req.Form.Get("done")
 	parentId := req.Form.Get("parentId")
 
+	var kase *cases.Case
```

Comment:
Why do we define kase and err here, rather than using the := notation inside the if/else?

Reply:
Because the kase might be either populated from Create or Update (its a shared var)